c_glib/README.md

```diff
@@ -142,6 +142,18 @@ $ meson compile -C c_glib.build
 $ sudo meson install -C c_glib.build
 ```
 
+> [!WARNING]
+>
+> When building GLib, it typically uses the Arrow library installed via brew. However, this can lead to build failures
+> if there are mismatches between the changes in Arrow's GLib and CPP libraries. To resolve this, you may need to
+> reference the Arrow CPP library built locally. In such cases, use the `-Darrow_cpp_build_dir`
```

Comment:
```suggestion
> When building Arrow GLib, it typically uses the Arrow C++ installed via Homebrew. However, this can lead to build failures
> if there are mismatches between the changes in Arrow's GLib and C++ libraries. To resolve this, you may need to
> reference the Arrow C++ library built locally. In such cases, use the `-Darrow_cpp_build_dir`
```